source/features/align-issue-labels.tsx

```diff
@@ -1,8 +1,56 @@
 import './align-issue-labels.css';
 
+import * as pageDetect from 'github-url-detection';
+import batchedFunction from 'batched-function';
+
 import features from '../feature-manager.js';
+import observe from '../helpers/selector-observer.js';
+
+const issueRowSelector = '[class^="IssueRow-module__row"]';
+const badgesContainerSelector = '[class*="trailingBadgesContainer"]';
+const mainContentInnerSelector = '[class^="MainContent-module__inner"]';
+const descriptionContainerSelector = '[class^="Description-module__container"]';
+const issueTypeContainerSelector = '[class*="issueTypeTokenContainer"]';
+
+function alignBadges(issueRows: HTMLElement[]): void {
+	for (const issueRow of issueRows) {
+		const badgesContainer = issueRow.querySelector(badgesContainerSelector);
+		if (!badgesContainer) {
+			continue;
+		}
+
+		const mainContentInner = issueRow.querySelector(mainContentInnerSelector);
```

Comment:
All querySelector usage should be replaced with select-dom. Import the "strict" version and then use `$()`

`$optional()` can also be used if it *already* an element doesn't exist. If some must exist now but doesn't in the future, `$()` throws an error so we know when/where the feature failed.